public/js/routing/router.js

```diff
@@ -64,10 +64,14 @@ export default class Router {
     this._updateContent();
   }
 
+  _isLeftClickWithoutModifiers(e) {
+    return e.button === 0 && !e.ctrlKey && !e.metaKey && !e.shiftKey && !e.altKey;
+  }
+
   _takeOverAnchorLinks(root) {
     root.querySelectorAll('a').forEach(element => {
       element.addEventListener('click', e => {
-        if (e.button !== 0 || e.ctrlKey || e.metaKey || e.shiftKey || e.altKey) {
+        if (!this._isLeftClickWithoutModifiers(e)) {
```

Comment:
suggestion: rename to _isLeftClick**With**Modifiers - this way you don't need the `!` and the predicate becomes a lot more readable.